### DISCUSSION TITLE
How to Limit or Restrict Number of Open code-server Tabs, Windows or Other Connections?

### DISCUSSION THREAD

**Discussion by doncadavona** [coder/code-server#6399] [Q&A] [answered]
To conserve some server and network resources, I wish to set a limit on the number of open tabs, windows or connections on a code-server instance. Is there an official way to do it? How?

Say I open 1 browser tab for `localhost:8080`. 
If I open another tab for `localhost:8080`, without closing the first tab, I should get a notice, or redirect me to the first tab.

Thank you in advance :)

**Comment by code-asher** [comment, marked as answer]
There is no built-in mechanism for doing this currently, as far as I know.

**Reply by doncadavona** [reply]
Alright, thank you anyways! :)
